app/errors.js

```diff
@@ -14,3 +14,6 @@ exports.validationError = message => internalError(message, exports.VALIDATION_E
 
 exports.AUTHENTICATION_ERROR = 'authentication_error';
 exports.authenticationError = message => internalError(message, exports.AUTHENTICATION_ERROR);
+
+exports.NOT_FOUND_ERROR = 'not_found_error';
```

Comment:
You need to add the mapping to the proper code here https://github.com/wolox-training/gcu-express-js/blob/351db3b71358e4a39e16d40dae6e84af282950f8/app/middlewares/errors.js#L6